processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -743,12 +745,13 @@ private LongColumnSerializer setupTimeWriter(final IOPeon ioPeon) throws IOExcep
   {
     ArrayList<GenericColumnSerializer> metWriters = Lists.newArrayListWithCapacity(mergedMetrics.size());
     final CompressedObjectStrategy.CompressionStrategy metCompression = indexSpec.getMetricCompressionStrategy();
+    final CompressionFactory.LongEncodingFormat metEncoding = indexSpec.getMetricLongEncodingFormat();
```

Comment:
This is more like a `longEncoding` than a `metEncoding` (keep in mind we will have long dimensions soon; also, `__time` is a long).